packages/liveblocks-react/src/comments/CommentsRoom.ts

```diff
@@ -80,7 +105,7 @@ function createOptimisticId(prefix: string) {
   return `${prefix}_${nanoid()}`;
 }
 
-export type RoomThreads<TThreadMetadata extends BaseMetadata> =
+export type State<Data = any> =
```

Comment:
Given that this is only for one use-case, I don't know if we should make all types generic like SWR.